src/libraries/Directory.Build.props

```diff
@@ -360,6 +359,20 @@
   <!-- Import it at the end of the props file to override the OutputPath for reference assemblies and use common directory props -->
   <Import Project="$(RepositoryEngineeringDir)referenceAssemblies.props" />
 
+  <PropertyGroup>
+    <DisableProjectRestore Condition="'$(MSBuildProjectExtension)' == '.depproj' or '$(MSBuildProjectExtension)' == '.pkgproj'">true</DisableProjectRestore>
+  </PropertyGroup>
+
+  <Import Project="$(RepositoryEngineeringDir)restore\repoRestore.props" Condition="'$(DisableProjectRestore)' == 'true'" />
+
+  <ItemGroup Condition="'$(IsSourceProject)' == 'true' or '$(IsReferenceAssembly)' == 'true' or '$(IsPartialFacadeAssembly)' == 'true'">
+    <PackageReference Include="Microsoft.DotNet.Build.Tasks.Packaging" Version="$(MicrosoftDotNetBuildTasksPackagingVersion)" PrivateAssets="all" IsImplicitlyDefined="true" />
+    <PackageReference Include="Microsoft.DotNet.GenFacades" Version="$(MicrosoftDotNetGenFacadesVersion)" PrivateAssets="all" IsImplicitlyDefined="true" />
```

Comment:
We might be able to condition this one on IsPartialFacadeAssembly.

Reply:
I had to revert this change as ApiCompat failed for System.IO.Pipes.AccessControl. Possibly because of https://github.com/dotnet/runtime/blob/master/src/libraries/System.IO.Pipes.AccessControl/src/System.IO.Pipes.AccessControl.csproj#L5. I will attempt this in a follow-up PR again.